Makefile

```diff
@@ -13,17 +13,20 @@ endif
 
 ### DIRECTORIES ###
 SRC_DIR     := src
+CMDS_DIR	:= commands
 INCLD_DIR   := include
 OBJS_DIR    := objs
 
 ### FILES ###
 SRC         :=	main.cpp \
 	Client.cpp \
 	ClientSocket.cpp \
+	CommandRegistry.cpp \
 	Server.cpp \
 	ServerSocket.cpp \
 	Socket.cpp \
-	SocketObserver.cpp
+	SocketObserver.cpp \
+	commands/PingCommand.cpp
```

Comment:
It is better to create an "add prefix list with $CMDS_DIR variable" for the futures commands

Reply:
we tried and did not succeed
we figured we'd let you show us how you'd approach the issue